change/@fluentui-react-dialog-c4b3b805-3dcc-4caa-9c9e-ceb9a1580a1a.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29391&commitId=1e183e3c127e90743c98f25d0bca5c7798277e84&env=prod&clientType=FLUENTUI)


  <details><summary><b>Tree</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tree.default open tree.chromium.png|871| Changed|
|Tree.default open tree - High Contrast.chromium.png|1174| Changed|
|Tree.default open tree - RTL.chromium.png|876| Changed|
|Tree.persona - Dark Mode.chromium.png|492| Changed|
|Tree.default open tree - Dark Mode.chromium.png|791| Changed|
|Tree.persona.chromium.png|653| Changed|
|Tree.persona - RTL.chromium.png|653| Changed|
|Tree.persona - High Contrast.chromium.png|901| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>